### DISCUSSION TITLE
Typst: Unable to fit mermaid diagram into column

### DISCUSSION THREAD

**Discussion by jkub6** [quarto-dev/quarto-cli#8542] [Q&A] [answered]
### Description

I have been unable to find a way to get my figure to fit within its column in a 2 column Typst document. Is there a way to auto shrink-to-fit figures? (Or is this a bug with Typst in Quarto?)

## Example

![image](https://github.com/quarto-dev/quarto-cli/assets/30987328/0dbb4e38-2c69-4a60-ace2-6dc441b52b97)

````qmd
---
title: 2 Column Layout
format:
  typst:
    columns: 2
---


## Heading 1

Some text here. This should wrap when it hits the column.

```{mermaid}
graph LR
  A---B---C---D---E---F
```

```{mermaid}
graph LR
  A---B---C---D---E---F---G---H---I---J
```

More text here. This should also wrap when it hits the column.
````

**Comment by cderv** [comment, marked as answer]
Thanks for the report. I think this is an issue and it is not handled. The size is set without adapting to column. We'll see if we can fix 
* https://github.com/quarto-dev/quarto-cli/issues/8543
